src/instructlab/chat/chat.py

```diff
@@ -482,6 +482,13 @@ def chat_cli(
         timeout=DEFAULT_CONNECTION_TIMEOUT,
         http_client=httpx.Client(cert=cert, verify=verify),
     )
+    # ensure the model specified exists on the server. with backends like vllm, this is crucial.
+    if not any(model == m.id for m in client.models.list().data):
+        if not os.path.exists(model):
```

Comment:
This check won't work if we're talking to a remote model endpoint. It's also making assumptions about how the model is named, which may not be a path to the model at all. That's how it works with our built-in llama-cpp, but it doesn't have to be that way.

Reply:
I think one of the other points of the PR is forcing the model to be a path, since we could have duplicate model names. 

Can we even talk to a remote model endpoint at this stage? Doesn't it have to be a local file?

Reply:
> I think one of the other points of the PR is forcing the model to be a path, since we could have duplicate model names.
> 
> Can we even talk to a remote model endpoint at this stage? Doesn't it have to be a local file?

It does not.  You can do `ilab chat --endpoint-url https://anything.openai.compatible/v1`.

Here's a suggested alternative:

Add:

```
DEFAULT_MODEL_OLD = "merlinite-7b-lab-Q4_K_M"
```

and then:
```suggestion
        if model == DEFAULT_MODEL_OLD:
            ... friendly message about using an old config that we know is now broken with instructions on how to fix it ...
```

Reply:
Ok, added this